internal/pkg/flink/internal/store/store_utils.go

```diff
@@ -171,36 +169,59 @@ func parseSetStatement(statement string) (string, string, error) {
 		}
 	}
 
-	if keyValuePair[0] != "" && keyValuePair[1] == "" {
+	keyWithQuotes := strings.TrimSpace(keyValuePair[0])
+	valueWithQuotes := strings.TrimSpace(keyValuePair[1])
+
+	if keyWithQuotes != "" && valueWithQuotes == "" {
 		return "", "", &types.StatementError{
 			Message:    "value for key not present",
 			Suggestion: `if you want to reset a key, use "RESET 'key'"`,
 		}
 	}
 
-	if keyValuePair[0] == "" && keyValuePair[1] != "" {
+	if keyWithQuotes == "" && valueWithQuotes != "" {
 		return "", "", &types.StatementError{
 			Message: "key not present",
 			Usage:   []string{"SET 'key'='value'"},
 		}
 	}
 
-	if keyValuePair[0] == "" && keyValuePair[1] == "" {
+	if keyWithQuotes == "" && valueWithQuotes == "" {
 		return "", "", &types.StatementError{
 			Message: "key and value not present",
 			Usage:   []string{"SET 'key'='value'"},
 		}
 	}
 
-	if !strings.HasPrefix(keyValuePair[0], "'") || !strings.HasSuffix(keyValuePair[0], "'") ||
-		!strings.HasPrefix(keyValuePair[1], "'") || !strings.HasSuffix(keyValuePair[1], "'") {
+	if !strings.HasPrefix(keyWithQuotes, "'") || !strings.HasSuffix(keyWithQuotes, "'") ||
+		!strings.HasPrefix(valueWithQuotes, "'") || !strings.HasSuffix(valueWithQuotes, "'") {
 		return "", "", &types.StatementError{
-			Message: "key and value must be enclosed by single quotes ''",
+			Message: "key and value must be enclosed by single quotes (')",
 			Usage:   []string{"SET 'key'='value'"},
 		}
 	}
 
-	return strings.ReplaceAll(keyValuePair[0], "'", ""), strings.ReplaceAll(keyValuePair[1], "'", ""), nil
+	// remove quotes
+	key := keyWithQuotes[1 : len(keyWithQuotes)-1]
+	value := valueWithQuotes[1 : len(valueWithQuotes)-1]
```

Comment:
I think `gofmt` should catch this, but:
```suggestion
	key := keyWithQuotes[1:len(keyWithQuotes)-1]
	value := valueWithQuotes[1:len(valueWithQuotes)-1]
```

Reply:
My GoLand does this automatically when saving (and uses gofmt in the background). Running gofmt explicitly formats it that way too. Only when I omit the `-1` does it remove the spaces:
`keyWithQuotes[1:len(keyWithQuotes)]`

Reply:
Same behavior on VS Code using gofmt as formatter